openedx/core/djangoapps/content_staging/tests/test_clipboard.py

```diff
@@ -364,7 +364,7 @@ def test_no_stealing_clipboard_content(self):
         response = nonstaff_client.get(olx_url)
         assert response.status_code == 403
 
-    def assertXmlEqual(self, xml_str_a: str, xml_str_b: str) -> bool:
+    def assertXmlEqual(self, xml_str_a: str, xml_str_b: str):
```

Comment:
Why did you need to get rid of this `-> bool` ?

Reply:
The function doesn't actually have a return statement so the type check was complaining about it.

Reply:
Oh, I see. You're right, it's not returning anything :p

Reply:
Although it would have been better to use `-> None` to specify that explicitly. No big deal though. Thanks for the fix!